src/StorageSync/StorageSync.Test/ScenarioTests/ServerEndpointTests.ps1

```diff
@@ -260,4 +260,203 @@ function Test-ServerEndpoint
+        $job = New-AzStorageSyncCloudEndpoint -ParentObject $syncGroup -Name $cloudEndpointName -StorageAccountResourceId $storageAccountResourceId -AzureFileShareName $azureFileShareName -StorageAccountTenantId $StorageAccountTenantId -AsJob 
+        $job | Wait-Job
+        $cloudEndpoint = get-job -Id $job.Id | receive-job -Keep
+        $cloudEndpoint | Set-AzStorageSyncCloudEndpointPermission
+
+        Write-Verbose "Resource: $serverEndpointName | Loc: $resourceLocation"
+        $job = New-AzStorageSyncServerEndpoint -ResourceGroupName $resourceGroupName -StorageSyncServiceName $storageSyncServiceName -SyncGroupName $syncGroupName -Name $serverEndpointName -ServerResourceId $registeredServer.ResourceId -ServerLocalPath $serverLocalPath -CloudTiering -VolumeFreeSpacePercent $volumeFreeSpacePercent -TierFilesOlderThanDays $tierFilesOlderThanDays -InitialUploadPolicy $initialUploadPolicy -Verbose -AsJob 
+        $job | Wait-Job
+        $serverEndpoint = get-job -Id $job.Id | receive-job -Keep
+        $serverEndpoint | Set-AzStorageSyncServerEndpointPermission
+
+        Write-Verbose "Removing ServerEndpoint: $serverEndpointName"
+        Remove-AzStorageSyncServerEndpoint -Force -ResourceGroupName $resourceGroupName -StorageSyncServiceName $storageSyncServiceName -SyncGroupName $syncGroupName -Name $serverEndpointName -AsJob | Wait-Job
+
+        Write-Verbose "Unregister Server: $($registeredServer.ServerId)"
+        Unregister-AzStorageSyncServer -Force -ResourceGroupName $resourceGroupName -StorageSyncServiceName $storageSyncServiceName -ServerId $registeredServer.ServerId -AsJob | Wait-Job
+
+        Write-Verbose "Removing CloudEndpoint: $cloudEndpointName"
+        Remove-AzStorageSyncCloudEndpoint -Force -ResourceGroupName $resourceGroupName -StorageSyncServiceName $storageSyncServiceName -SyncGroupName $syncGroupName -Name $cloudEndpointName -AsJob | Wait-Job
+
+        Write-Verbose "Removing SyncGroup: $syncGroupName"
+        Remove-AzStorageSyncGroup -Force -ResourceGroupName $resourceGroupName -StorageSyncServiceName $storageSyncServiceName -Name $syncGroupName -AsJob | Wait-Job
+
+        #Start-Sleep -Seconds 300
+        Write-Verbose "Removing StorageSyncService: $storageSyncServiceName"
+        Remove-AzStorageSyncService -Force -ResourceGroupName $resourceGroupName -Name $storageSyncServiceName -AsJob | Wait-Job
+
+        if(IsLive)
+        {
+            Write-Verbose "Removing: $AzureFileShareName | Loc: $resourceLocation | Type : AzureStorageShare"
+            Remove-StorageShare -Name $AzureFileShareName -Context $context | Out-Null
+        }
+
+        Write-Verbose "Removing $StorageAccountName | Loc: $resourceLocation | Type : StorageAccount"
+        Remove-AzStorageAccount -ResourceGroupName $resourceGroupName -Name $StorageAccountName
+    }
+    finally
+    {
+        # Cleanup
+        Write-Verbose "Removing ResourceGroup : $resourceGroupName"
+        Clean-ResourceGroup $resourceGroupName
+    }
```

Comment:
is there coverage of the Add/Remove SEP RBAC

Reply:
Yes, Its in one of the test ps1 file. Check Test-ServerEndpointAfterIdentityMigration